src/validation.cairo

```diff
@@ -206,9 +208,22 @@ fn validate_coinbase(block: @Block, total_fees: u256) -> Result<(), ByteArray> {
     Result::Ok(())
 }
 
+// Return BTC reward => pow to 8 to transform into Sats. Otherwise difficult to cast to u64 correctly after if needed
+fn compute_block_reward(block_height: u32) -> Result<u64, ByteArray> {
+    let number_halvings = block_height / 210_000;
+    match number_halvings {
+        0 => { return Result::Ok(REWARD_INITIAL.try_into().unwrap()); }, // return REWARD_INITAL
+        _ => {}
+    }
+    let current_reward = shr(REWARD_INITIAL, number_halvings);
```

Comment:
Reward should be in sats. Here REWARD_INITIAL is 50.

Reply:
Change to return a SATS amount, added the shl to do it